dependencies.py

```diff
@@ -42,35 +42,36 @@
 
 #: List of expected dependencies.
 DEPENDENCIES = [
-    Dependency("SQLAlchemy", "sqlalchemy", "<1.4"),
-    Dependency("GeoAlchemy2", "geoalchemy2", "==0.10.2"),
+    Dependency("SQLAlchemy", "sqlalchemy", "==2.0.6"),
+    Dependency("GeoAlchemy2", "geoalchemy2", "==0.13.*"),
     Dependency("lizard-connector", "lizard_connector", "==0.7.3"),
-    Dependency("pyqtgraph", "pyqtgraph", ">=0.11.1,<0.12"),
-    Dependency("threedigrid", "threedigrid", ">=1.2.3"),
-    Dependency("cached-property", "cached_property", ""),
-    Dependency("threedi-schema", "threedi_schema", "==0.214.4"),
-    Dependency("threedi-modelchecker", "threedi_modelchecker", "==1.0.*"),
-    Dependency("threedidepth", "threedidepth", "==0.4"),
+    Dependency("pyqtgraph", "pyqtgraph", ">=0.13.2"),
+    Dependency("threedigrid", "threedigrid", ">=2.0.3"),
+    Dependency("threedi-schema", "threedi_schema", ">=0.216.2"),
```

Comment:
@arjanverkerk deze moeten we wel binnen, in elk geval op .216. Ik stel voor  `"==0.216.4"`